src/devices/zemismart.ts

```diff
@@ -368,6 +368,43 @@ const definitions: Definition[] = [
         description: 'Smart 20A outlet',
         extend: [identify(), tuya.modernExtend.tuyaOnOff({indicatorMode: true, onOffCountdown: true, childLock: true})],
     },
+    {
+        fingerprint: tuya.fingerprint('TS004F', ['_TZ3000_11pg3ima']),
+        model: 'ZMR4',
+        vendor: 'Zemismart',
+        description: 'Wireless switch with 4 buttons',
```

Comment:
```suggestion
        description: 'Wireless switch with 4 buttons',
        extend: [battery()],
```

And remove battery from exposes, and configure

Reply:
Runs well, thanks.